test/lib/pgBackRestTest/Common/LogTest.pm

```diff
@@ -1,492 +0,0 @@
-####################################################################################################################################
```

Comment:
logTest.pm has been deleted but there is a reference at line 1862 for test/lib/pgBackRestTest/Common/LogTest.pm in code-count/file-type.yaml

Reply:
This file is auto-generated with every release, but no harm in removing that file now. Done in 299158379.